content/docs/guides/upgrading.md

```diff
@@ -21,8 +21,6 @@ Essentially, this means that:
 # Changelog
 You can read the history file to learn about the changes in every release:
 
-[cucumber changelog](https://github.com/cucumber/cucumber/blob/master/History.md)
```

Comment:
When removing this, the text before probably has to change, too - otherwise it's very confusing.

In addition - can we find a place where changes in releases are presented? This page is a bit rough - https://github.com/cucumber/cucumber/releases - and there's no **shared** changelog in the cucumber/cucumber repo, but each subdirectory has a changelog, such as https://github.com/cucumber/cucumber/blob/master/cucumber-react/CHANGELOG.md 

Hope this helps!